units/commit_queue.go

```diff
@@ -609,7 +609,7 @@ func AddMergeTaskAndVariant(patchDoc *patch.Patch, project *model.Project, proje
 	project.Tasks = append(project.Tasks, mergeTask)
 	project.TaskGroups = append(project.TaskGroups, mergeTaskGroup)
 
-	validationErrors := validator.CheckProjectSyntax(project, true)
+	validationErrors := validator.CheckProjectErrors(project, true)
 	validationErrors = append(validationErrors, validator.CheckProjectSettings(project, projectRef)...)
 	catcher := grip.NewBasicCatcher()
 	for _, validationErr := range validationErrors.AtLevel(validator.Error) {
```

Comment:
II know you didn't write this but I think we can just do `catcher.Extend(validationErrors.AtLevel(validator.Error))` instead of looping, or can just use the ValidationErrorsToString functioin that's used elsewhere

Reply:
I got an error when I replaced the loop with `catcher.Extend(validationErrors.AtLevel(validator.Error))`. It error pretty much said `catcher.Extent()` expects an `[]error` and could not accept `validator.ValidatorErrors`